src/CONST.js

```diff
@@ -918,8 +923,8 @@ const CONST = {
 
         WEBSITE: /^((https?|ftp):\/\/)(([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}(:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(#[-a-z\d_]*)?$/i,
 
-        // eslint-disable-next-line max-len, no-misleading-character-class
-        EMOJIS: /[\p{Extended_Pictographic}\u200d\u{1f1e6}-\u{1f1ff}\u{1f3fb}-\u{1f3ff}\u{e0020}-\u{e007f}\u20E3\uFE0F]|[#*0-9]\uFE0F?\u20E3/gu,
+        // eslint-disable-next-line no-misleading-character-class
+        EMOJIS: new RegExp(`${EMOJIS.source}`, 'g'),
```

Comment:
<img width="552" alt="Screenshot 2023-04-20 at 16 27 06" src="https://user-images.githubusercontent.com/129500732/233323718-9047a9d3-abf9-4c80-af85-573c5d14ca58.png">
The EMOJIS REGEX is the same as the old one